packages/augur-core/source/contracts/Augur.sol

```diff
@@ -93,6 +93,8 @@ contract Augur is IAugur, IAugurCreationDataGetter, CashSender {
 
     int256 private constant DEFAULT_MIN_PRICE = 0;
     int256 private constant DEFAULT_MAX_PRICE = 1 ether;
+    uint256 private constant RECOMMENDED_TRADE_INTERVAL = 10;
+    uint256 private constant DEFAULT_RECOMMENDED_TRADE_INTERVAL = 10 ** 17;
```

Comment:
Remove the "Recommended" bit from this.  From what I can tell these are hard coded values that cannot be changed, so there is nothing about them that makes them just a "recommendation".

Reply:
I labeled it this way since this is in the core contracts here where this isn’t enforced in any way, but is a recommended value for use by external trading contracts

Reply:
Oh, I just noticed that this is in the core contracts.  Can we make it so that this value is a market input parameter instead of hard coded?  The UI can then fix it, and optionally the UI could even hide any market that doesn't set it to a reasonable value, but that way we _can_ change it later.

From the point of view of the core contracts, this value doesn't prevent the market from being valid and reportable, so I don't think it should be hard coded in the core contracts.

Reply:
We don't want it to be configurable really and because of scalars it will be variable. The purpose of doing it at market creation like this is to save gas since there isnt really a registration of markets in the trading contracts so we'd have to at a minimum waste gas checking if it has been precalculated.